docker-setups/angular-docker.md

```diff
@@ -205,7 +205,9 @@ services:
     image: angular-sample-image
     container_name: angular-sample-container
     ports:
-      - '127.0.0.1:5000:80'
+      - '127.0.0.1:5000:5000' # Map the ports properly for security
```

Comment:
The previous one was fine as well. What we are looking for here is that we should add the `127.0.0.1` address so that ports are only exposed locally. You can update the comment accordingly.

Reply:
Reverted back to 127.0.0.1:5000:80 and updated the comment.